### ISSUE TITLE
Internationalization break generation of docs

### ISSUE THREAD

**Issue by flozero** [f3ltron/vuepress-plugin-docgen#30]
The generation of the doc create folder and file no matter of locales.

Locales in vuepress need /zh/ etc subfolder if local is present. Effectively i didnt think about that before.

Easy to improve just add a local param and i have to change the generation and add more generation if local is present and change the sidebar override

**Comment by zhaozhe0831** [comment]
Do you have any feature or TODO list ?
I'm interested about the plugin.
Maybe I could do something to make the plugin better ~

**Comment by flozero** [comment]
Hello @zhaozhe0831  thanks for appreciate my plugni :). I don't have much time to take care of it yeah ... 

But in my todo list ii wanted to dooo:
- upgrade to last vuepress
- improve design
- add initernationalisation
- hot or hard reload when change is done in dev

If you want to work on it tell me I can add those tasks and assign them to you and I will try to help you as much as possible for sure

**Comment by zhaozhe0831** [comment]
- [x] upgrade to last vuepress
- [ ] improve design
- [ ] add initernationalisation
- [ ] hot or hard reload when change is done in dev

Isn't it support the latest vuepress now? your mean Vue 3.0 version ？

initernationalisation？I think maybe I can help chinese version

**Comment by zhaozhe0831** [comment]
Maybe I could add feature.

allow user DIY components introduction page.

![image](https://user-images.githubusercontent.com/12945873/93290133-a041fb80-f812-11ea-988b-27a1af4d7d51.png)

we can let user introduce their components in here. 

to parser a README.md in `src/components/README.md` for the introduction.

what do you think about my opinion ? If you think ok, I'll do it in my free time.

**Comment by flozero** [comment]
Yes, upgrade the version I am using i didn't test on the last vuepress if my plugins are working. I stopped working with vuepress core team for a while I have to admit.

Internationalization is when you want to use i18n for the documentation. But it's not the most important feature I think.

What do you mean by DIY. I have like template already

https://f3ltron.github.io/vuepress-plugin-docgen/guide/more.html#additionalpages

Tell me if it's what you are speaking about. But with that, you can override the default template rendering function and add yours. Maybe there is a lack of documentation on this

**Comment by flozero** [comment]
You mean the home page. I would have to check about it but normally you can override the introduction page of components too with the index or with a readme. Not 100 % sure.

And of course don't hesitate to ask I will try to guide you about the knowledge of the plugin! It's really cool to see more people using it

**Comment by zhaozhe0831** [comment]
Thank you!

**Comment by zhaozhe0831** [comment]
oh, yesterday I was a little busy for my work.
I will have 8 days off from 10.1 to 10.8 (October).
Maybe we can 'talk' about vuepress-plugin-docgen.
actually I'm not good at English... I afraid I can't understand the all meaning of your word.
I always use Google translate.
If I'm better at English, I would like to talk with you~
I can use WeChat, record , or online meeting ,or something else.
I really want to make easier for creating Components Document.
